web-app/js/unifina/streamr-chart/streamr-chart.js

```diff
@@ -70,7 +70,10 @@ function StreamrChart(parent, options) {
 		]
 
 		rangeConfig.forEach(function(c) {
-			var $option =  $("<option value='"+c.range+"'>"+c.name+"</option>")
+			var $option =  $('<option/>', {
```

Comment:
While manually testing, I noticed that the chart range dropdown initially shows an empty label. Changes look ok, so not sure if it's related to these changes or some earlier ones?

Reply:
#517 opened